dice_bag.gemspec

```diff
@@ -15,6 +15,8 @@ Gem::Specification.new do |s|
   s.require_paths = ['lib']
 
   s.add_dependency 'rake'
+  s.add_dependency 'thor', '~> 0'
```

Comment:
maybe ~> 0.0 is better because if version 1.0 is ever released may have breaking changes